biz.aQute.bnd.util/src/aQute/bnd/classfile/ClassFile.java

```diff
@@ -15,7 +15,7 @@
 import aQute.lib.io.IO;
 
 public class ClassFile extends ElementInfo {
-	public static final int		MAJOR_VERSION	= 61;
+	public static final int		MAJOR_VERSION	= 68;
```

Comment:
No. Only change this when there is a class file change you are supporting added by that major version. So leave as 61 (Java 17) for now.

Reply:
oh ok, sorry. reverted.